tests/websocket/test_streamer.py

```diff
@@ -39,7 +39,11 @@ def test_streamer(self) -> None:
             addresses.append(AddressItem(idx, wallet.get_address(wallet.get_key_at_index(idx))))
 
         # Create the expected result.
-        expected_result: list[dict[str, Any]] = [{'type': 'stream:history:begin', 'id': stream_id}]
+        expected_result: list[dict[str, Any]] = [{
+            'type': 'stream:history:begin',
+            'id': stream_id,
+            'sliding_window_size': 10000
```

Comment:
issue(blocking): This should be `window_size`.

`sliding_window_size` is the internal name, but the protocol extracts `window_size` from the client's message

Reply:
Done!